aws/cloudformation/components/alarms.yml.erb

```
@@ -27,6 +27,7 @@
       ComparisonOperator: GreaterThanThreshold
       Dimensions:
         - {Name: InstanceId, Value: !Ref <%=daemon%>}
+        - {Name: Environment, Value: <%= rack_env? %>}
```

I think this is supposed to be

```suggestion
        - {Name: Environment, Value: <%= rack_env %>}
```

Although looking at the other code in this template, should it maybe be

```suggestion
        - {Name: Environment, Value: <%= environment %>}
```